mdtraj/geometry/rdf.py

```diff
@@ -67,7 +70,10 @@ def compute_rdf(traj, pairs=None, r_range=None, bin_width=0.005,
         r_range = np.array([0.0, 1.0])
     r_range = ensure_type(r_range, dtype=np.float64, ndim=1, name='r_range',
                           shape=(2,), warn_on_cast=False)
-    bins = np.arange(r_range[0], r_range[1], bin_width)
+    if n_bins is not None:
```

Comment:
No worries hahah. I think what you proposed is simple enough. We do need to trust the user to know at least a little about what they're doing :)